### ISSUE TITLE
No longer able to connect my google account to GKeep List Sync using Master Token

### ISSUE THREAD

**Issue by shwarnock** [fcastilloec/gkeep-list-sync#22]
In the past, I have had to use the master token process in order to link my google account to the gkeep integration. However, starting this morning, I have been unable to connect my account even using this process.

**Comment by fcastilloec** [comment]
I've noticed this as well. The problem is not with this integration but with Home Assistant's latest update. The issue was introduced in `2024.6.1`.
There's a PR that will fix the issue (https://github.com/simon-weber/gpsoauth/pull/67) and once merged, we could fix this.
In the meantime, I downgraded to `2024.6.0` and this issue is no longer a problem. Use this command on the terminal:

```sh
ha core update --version 2024.6.0
```
or restore from a backup.
I'm going to leave this issue open, just so other people can find it
